portfolio/src/main/webapp/script.js

```diff
@@ -297,3 +311,72 @@ function addMarker(map, coordinates, title, icon, description){
     infoWindow.open(map, marker);
   });	
 }
+
+google.charts.load('current', {'packages':['line']});
+google.charts.setOnLoadCallback(drawTemperatureChart);
+
+/** Draws the Ithaca average monthly temperature chart. */
+function drawTemperatureChart() {
+
+  var data = new google.visualization.DataTable();
+  data.addColumn('string', 'Month');
+  data.addColumn('number', 'High');
+  data.addColumn('number', 'Low');
+
+  data.addRows(tempData);
+
+  const options = {
+    chart: {
+      title: 'Average Monthly Temperatures in Ithaca',
+      subtitle: 'in degrees Celcius',
+    },
+    series: {
+      0: { color: '#DC143C' },
+      1: { color: '#1E90FF' },
+    },
+    backgroundColor: '#FAEBD7',
+    width: 900,
+    height: 500
+  };
+
+  var chart = new google.charts.Line(document.getElementById('weather-chart-container'));
+  chart.draw(data, google.charts.Line.convertOptions(options));
+}
+
+google.charts.load('current', {'packages':['corechart']});
```

Comment:
I'm not super familiar with the Google charts APIs, but I wonder if you can combine this `google.charts.load` with the one above, like `google.charts.load('current', {'packages':['line', 'corechart']});`?

I think this might be a little more efficient (IIUC it would only make one request to the server to load).  Also with your current code I suspect `drawTemperatureChart` and `drawVoteChart` might each be called twice since `load` would complete twice (I'm not 100% sure, you might be able to verify this with Chrome Dev Tools)

Reply:
Yes, this works thanks, more efficient. Fixed